serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/Events.kt

```diff
@@ -131,6 +173,17 @@ object EventSerializer : JsonContentPolymorphicSerializer<Event>(Event::class) {
     }
 }
 
+object RoomMessageEventContentSerializer : JsonContentPolymorphicSerializer<RoomMessageEventContent>(RoomMessageEventContent::class) {
+    override fun selectDeserializer(element: JsonElement) = element.jsonObject["msgtype"]!!.jsonPrimitive.content.let { type ->
```

Comment:
This should be `element.jsonObject["msgtype"]?.jsonPrimitive?.content?.let` to account for redacted messages - I've got this locally and it fixes it - I can push it here and will mark resolved.